EIPS/eip-7932.md

```diff
@@ -182,8 +106,8 @@ Each type of algorithm MUST specify the following fields:
 
 | Field Name | Description |
 |-|-|
-|`ALG_TYPE`| The uint8 of the algorithm unique ID |
-|`MAX_SIZE`| The maximum size of `signature_info` field in a transaction |
+|`ALG_TYPE`| The uint8 that would be present at `signature_info[0]` |
+|`MAX_SIZE`| The maximum possible size of the `signature_info` field including the `ALG_TYPE` byte |
```

Comment:
Should MAX_SIZE be deferred to the signature validation? e.g., for secp256k1 there is an exact size (MIN_SIZE) as well.